packages/web/src/components/collection/mobile/CollectionHeader.tsx

```diff
@@ -185,8 +188,25 @@ const CollectionHeader = ({
     )
   }
 
+  const renderDogEar = () => {
+    // Omitting isOwner and hasStreamAccess to ensure we always show gated DogEars
```

Comment:
wait i thought the dog ears are supposed to go away after purchase?

Reply:
wait you're right... I copied this from track but I think we should start passing these args to both. Update incoming...